openstef_dbc/database.py

```diff
@@ -70,6 +75,10 @@ class DataBase(metaclass=Singleton):
     # predictions methods
     get_predicted_load = _predictions.get_predicted_load
     get_predicted_load_tahead = _predictions.get_predicted_load_tahead
+    get_prediction_including_components = (
+        _predictions.get_prediction_including_components
+    )
```

Comment:
Alternatively, you could specify this as a parameter in the 'get_precition(componets=True)' function. I don;t have a preference.